test/test_StatisticalInverseProblem/test_parallel_h5.sh.in

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -eu
+set -o pipefail
+
+have_mpi=@HAVE_MPI@
+have_hdf5=@HAVE_HDF5@
+
+if [ $have_mpi -eq 1 ]; then
+  if [ $have_hdf5 -eq 1 ]; then
+    PROG="mpirun -np 2 ./test_parallel_h5"
+
+    INPUT="${srcdir}/test_StatisticalInverseProblem/input_test_parallel_h5.txt"
+
+    $PROG $INPUT
+
```

Comment:
Would it be possible to use h5diff, h5ls, or ls (in descending order of preference) to make sure the output is sane before we trash it?